Mediapipe.Net/Framework/Format/ImageFrame.cs

```diff
@@ -25,7 +25,7 @@ public ImageFrame() : base()
 
         public ImageFrame(ImageFormat format, int width, int height) : this(format, width, height, DefaultAlignmentBoundary) { }
 
-        public ImageFrame(ImageFormat format, int width, int height, uint alignmentBoundary) : base()
+        public ImageFrame(ImageFormat format, int width, int height, uint alignmentBoundary)
```

Comment:
Hmmm. Why remove it?

Reply:
Wait I removed this by accident, let me re-add that.